diff --git a/node_modules/koa-body/lib/index.d.ts b/node_modules/koa-body/lib/index.d.ts
index a590789..ef20b8b 100644
--- a/node_modules/koa-body/lib/index.d.ts
+++ b/node_modules/koa-body/lib/index.d.ts
@@ -3,11 +3,5 @@ import type { Middleware } from 'koa';
 import * as Koa from 'koa';
 import type { Files } from 'formidable';
 export * from './types';
-declare module 'koa' {
-    interface Request extends Koa.BaseRequest {
-        body?: any;
-        files?: Files;
-    }
-}
 export declare function koaBody(options?: Partial<KoaBodyMiddlewareOptions>): Middleware;
 export default koaBody;
